functionary/ui/templates/partials/workflows/step_list.html

```diff
@@ -1,84 +1,77 @@
-                                type="button"
-                                title="Add Step Below"
-                                hx-target="body"
-                                hx-swap="beforeend"
-                                hx-get="{% url 'ui:workflowstep-create' workflow.pk %}"
-                                hx-vals='{"next": "{{ step.next.id }}"}'/>
-                        <button class="button is-small mr-2 is-white has-text-info fa fa-pencil-alt"
-                                type="button"
-                                title="Edit"
-                                hx-target="body"
-                                hx-swap="beforeend"
-                                hx-get="{% url 'ui:workflowstep-edit' workflow.pk step.pk %}"/>
-                        <button class="button is-small is-white has-text-danger fa fa-trash"
-                                type="button"
-                                title="Delete"
-                                hx-target="#workflow-steps"
-                                hx-swap="outerHTML"
-                                hx-delete="{% url 'ui:workflowstep-delete' workflow.pk step.pk %}"/>
-                    </td>
+<div id="workflow-steps" class="ps-4 py-2">
+    <!--TODO: A table is not really suitable for this data. This is a placeholder-->
+    <div id="tableSteps" class="table-container">
+        <table class="table table-striped table-sm">
+            <thead>
+                <tr>
+                    <th scope="col"></th>
+                    <th scope="col">Name</th>
+                    <th scope="col">Function</th>
+                    <th scope="col">Parameters</th>
+                    <th scope="col"></th>
                 </tr>
-            {% endfor %}
-        </tbody>
-    </table>
-    <button class="button is-small"
+            </thead>
+            <tbody id="steps">
+                {% for step in workflow.ordered_steps %}
+                    <tr id="step-{{ step.id }}">
+                        <td>
+                            {% with previous=step.previous next=step.next %}
+                                <div class="d-flex {% if previous is None %}justify-content-end{% elif next is None %}justify-content-start{% else %}justify-content-between{% endif %}">
+                                    <button class="btn btn-small text-primary fa fa-chevron-up border-0"
+                                            id="step-{{ step.id }}-move-up"
+                                            type="button"
+                                            title="Move Step Up"
+                                            {% if previous is None %}hidden{% endif %}
+                                            hx-post="{% url 'ui:workflowstep-move' workflow.pk step.pk %}"
+                                            hx-target="#workflow-steps"
+                                            hx-swap="OuterHTML"
```

Comment:
There's a typo / bug here that was there before, but just sort of happened to be masked.  This is case sensitive and needs to be `outerHTML`.  As is, the value isn't valid and reverts to the default of `innerHTML`, which results in the step list shifting a little bit when you first move a step.